libnopegl/src/atlas.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 Nope Project
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef ATLAS_H
+#define ATLAS_H
+
+#include <stdint.h>
+#include <stdlib.h>
+
+struct ngl_ctx;
+struct atlas;
+
+struct bitmap {
+    uint8_t *buffer;
+    size_t stride;
+    int32_t width, height;
```

Comment:
uint32_t ?
Not related but I will probably change the texture dimensions to uint32_t.

Reply:
We use `int32_t` everywhere for `width` and `height` so I'd rather keep that consistent

Reply:
Ok.